### PR TITLE
coredns: forward request as it-is when using host's resolver i.e `/etc/resolv.conf`

### PR DESCRIPTION
**Pull request by flouthoc**
Using trust-dns-resolver forces us to read ip , while we don't need to
do it since we should be acting as a proxy. Following PR makes sure that
we forward DNS response as it-is from host's resolver.

* Fixes the issue which was discussed where we were not honoring actual
TTL.

### CODE REVIEW

**Comment by flouthoc** [comment]
@mheon @Luap99 @baude @rhatdan This fixes the `TTL` issue which was reported yesterday and also removes `trust-dns-resolver` which is not solving our use-case.

File: src/dns/coredns.rs [hunk 1]
```diff
@@ -222,39 +242,24 @@ impl CoreDns {
...
-                                                                        .set_name(record_name.clone())
-                                                                        .set_ttl(86400)
-                                                                        .set_rr_type(RecordType::AAAA)
-                                                                        .set_dns_class(DNSClass::IN)
-                                                                        .set_rdata(RData::AAAA(ipv6))
-                                                                        .clone(),
-                                                                );
-                                                            }
-                                                        }
+                                        for nameserver in nameservers {
```
Comment (mheon) [inline src/dns/coredns.rs:248]: If all servers fail, do we respond at all?

Comment (flouthoc) [reply src/dns/coredns.rs:248]: @mheon It should time out. It was decided by everyone in the last PR.

Comment (mheon) [reply src/dns/coredns.rs:248]: Alright, seems reasonable.

**Comment by mheon** [comment]
Changes LGTM

**Comment by flouthoc** [comment]
@Luap99 @baude Please verify patch and merge me.

**Review by Luap99 — APPROVED** [review]
LGTM

**Pull request merged**